soc/alif/ensemble/e7/Kconfig.defconfig.ae722f80f55d5xx_rtss_he

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2025 Alif Semiconductor
+#
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_AE722F80F55D5XX_RTSS_HE
+
+config NUM_IRQS
+	default 480
```

Comment:
that's 480 function pointers = 480 * 4 = 1920 byes total. GIC is GIC!

Reply:
Problem is memory usage. If you are not aware, ISR table in zephyr is statically constructed using `gen_isr_tables.py` based on this number. If there is `IRQ_CONNECT` for a number, it will be bound to specific function address, otherwise it points to `z_irq_spurious` and filled. This sums to 1920 * 2 times. ~4K RAM is consumed always.